persistence/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/ReadSideImpl.scala

```diff
@@ -71,23 +68,32 @@ private[lagom] class ReadSideImpl @Inject() (
         case None      => throw new IllegalArgumentException(s"ReadSideProcessor ${clazz.getName} returned 0 tags")
       }
 
-      val globalPrepareTask = ClusterStartupTask(
-        system, s"readSideGlobalPrepare-$encodedReadSideName",
-        () => processorFactory().buildHandler().globalPrepare().toScala,
-        config.globalPrepareTimeout, config.role, config.minBackoff, config.maxBackoff, config.randomBackoffFactor
-      )
-
-      val processorProps = ReadSideActor.props(processorFactory, registry.eventStream[Event], eventClass, globalPrepareTask, config.globalPrepareTimeout)
-
-      val backoffProps = BackoffSupervisor.propsWithSupervisorStrategy(
-        processorProps, "processor", config.minBackoff, config.maxBackoff, config.randomBackoffFactor, SupervisorStrategy.stoppingStrategy
-      )
+      val globalPrepareTask: ClusterStartupTask =
+        ClusterStartupTask(
+          system,
+          s"readSideGlobalPrepare-$encodedReadSideName",
+          () => processorFactory().buildHandler().globalPrepare().toScala,
+          config.globalPrepareTimeout,
+          config.role,
+          config.minBackoff,
+          config.maxBackoff,
+          config.randomBackoffFactor
+        )
+
+      val readSideProps =
+        ReadSideActor.props(
+          config,
+          eventClass,
+          globalPrepareTask,
+          registry.eventStream[Event],
+          processorFactory
+        )
```

Comment:
There were 3 steps, now there's only two because the BackoffSupervision is inside the actor.